crypto/keys/bcrypt/base64.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2011 The Go Authors. All rights reserved.
```

Comment:
we should open an issue on how/if we can remove this file and use the canonical version

Reply:
I agree. I need to look at it more as well, as CI is not happy with this file too (there were no CI in tendermint/crypto).
I wonder if we should enhance it, or leave it as is and exclude it from CI.

Reply:
Relevant: https://github.com/tendermint/tendermint/pull/3027, TM don't use their own fork.